Documentation/Books/Manual/Foxx/Users.md

```diff
@@ -27,18 +27,30 @@ Then create the setup script with the following content:
 ```js
 'use strict';
 const db = require('@arangodb').db;
-const sessions = module.context.collectionName('sessions');
-const users = module.context.collectionName('users');
-
-if (!db._collection(sessions)) {
-  db._createDocumentCollection(sessions);
+const documentCollections = ['sessions','users'];
+const edgeCollections = [];
+
+for (const localName of documentCollections) {
+  const qualifiedName = module.context.collectionName(localName);
+  if (!db._collection(qualifiedName)) {
+    db._createDocumentCollection(qualifiedName);
+  } else if (module.context.isProduction) {
+    console.warn(`collection ${qualifiedName} already exists. Leaving it untouched.`)
```

Comment:
The generated setups actually use `console.debug` (which isn't logged by default so it results in less noise). So if you want to reflect the generator output, you should probably use that.